python_modules/dagster/dagster_tests/asset_defs_tests/partition_mapping_tests/test_time_window_partition_mapping.py

```diff
@@ -331,13 +372,6 @@ def test_daily_to_daily_many_to_one():
             ["2022-12-30"],
             datetime(2022, 12, 31, 1),
         ),
-        (
-            DailyPartitionsDefinition(start_date="2022-01-01"),
```

Comment:
This tests an invalid situation: the set of partition keys that we're mapping from includes partitions that are not in the `PartitionsDefinition` that we're mapping from.

Ideally, we'd add some sort of validation to raise an error when this illegal input is provided to the partition-mapping machinery, but I avoided doing so here out of concern that performing the validation would have perf implications.